### DISCUSSION TITLE
Using Chroma within Flowise

### DISCUSSION THREAD

**Discussion by cfa532** [FlowiseAI/Flowise#286] [Q&A] [answered]
I am trying to use Chroma as vector store. I do not have any data in Chroma yet, so I use Chroma Upsert Doc as vector store retriever. I left the Chroma URL blank for it shall be integrated into the LangChain. I got an Error: Request failed with status code 400. It seems I get to run an instance of Chroma as server, doesn't it?

**Comment by cfa532** [comment]
I figured it out. Pull Chroma code from Github and run docker compose. Now I get an instance of Chroma up and running locally, and get another error message immediately, Error: Chroma getOrCreateCollection error: Error: TypeError: fetch failed

However my new question is, how should I debug the flow? where to look for the trace messge?

**Reply by HenryHengZJ** [reply, marked as answer]
if you are running both Flowise and Chroma on docker, it wouldnt work for now.

here's the docs - https://docs.flowiseai.com/vector-stores/chroma

**Reply by cfa532** [reply]
Thanks, but it does not work for me on MacOS. However, following the instructions there, I run a local copy of Flowise and a docker version of Chroma. It finally worked.
